### DISCUSSION TITLE
Passing data to Hovercard

### DISCUSSION THREAD

**Discussion by sewalsh** [chakra-ui/zag#564] [Q&A]
I'd like to enable hover cards on links.

What's the recommended way of passing data from each link so that I can access it within the hover card?

**Comment by anubra266** [comment]
@sewalsh I'm not sure I understand your question, could you shed more light. Also an example would help

**Comment by segunadebayo** [comment]
Closing as outdated — the API has been significantly reworked since this was posted.
